src/main.jsx

```diff
@@ -2,11 +2,12 @@ import ReactDOM from 'react-dom/client'
 import { BrowserRouter } from 'react-router-dom'
 
 import App from './App.jsx'
+import React from 'react'
 
 ReactDOM.createRoot(document.getElementById('root')).render(
-  // <React.StrictMode>
-  <BrowserRouter>
-    <App />
-  </BrowserRouter>,
-  // </React.StrictMode>,
+  <React.StrictMode>
```

Comment:
the previous commit comment mentioned this was to remove double rendering, but that is there on the dev server intentionally to make bugs obvious so that we catch them.